backend/src/models/teacher.js

```diff
@@ -1,82 +1,83 @@
-const { Model } = require('objection')
-const tableTeachers = 'teachers'
+const { Model } = require("objection");
+const tableTeachers = "teachers";
 
 class Teacher extends Model {
-  static get tableName () {
-    return tableTeachers
+  static get tableName() {
+    return tableTeachers;
   }
 
-  static get idColumn () {
-    return ['TeacherID']
+  static get idColumn() {
+    return ["TeacherID"];
   }
 
-  static get relationMappings () {
-    const NativeLanguage = require('./nativeLanguage')
-    const Status = require('./status')
-    const StatusUpdate = require('./statusUpdate')
+  static get relationMappings() {
+    const NativeLanguage = require("./nativeLanguage");
+    const Status = require("./status");
+    const StatusUpdate = require("./statusUpdate");
 
     return {
       nativeLanguage: {
         relation: Model.BelongsToOneRelation,
         modelClass: NativeLanguage.model,
         join: {
-          from: 'teachers.NativeLanguageID',
-          to: 'nativeLanguages.NativeLanguageID'
-        }
+          from: "teachers.NativeLanguageID",
+          to: "nativeLanguages.NativeLanguageID",
+        },
       },
 
       secondLanguage: {
         relation: Model.BelongsToOneRelation,
         modelClass: NativeLanguage.model,
         join: {
-          from: 'teachers.SecondLanguageID',
-          to: 'nativeLanguages.NativeLanguageID'
-        }
+          from: "teachers.SecondLanguageID",
+          to: "nativeLanguages.NativeLanguageID",
+        },
       },
 
       status: {
         relation: Model.BelongsToOneRelation,
         modelClass: Status.model,
         join: {
-          from: 'teachers.StatusID',
-          to: 'statuses.StatusID'
-        }
+          from: "teachers.StatusID",
+          to: "statuses.StatusID",
+        },
       },
 
       statusUpdates: {
         relation: Model.HasManyRelation,
         modelClass: StatusUpdate.model,
         join: {
-          from: 'teachers.TeacherID',
-          to: 'statusUpdates.TeacherID'
-        }
-      }
-    }
+          from: "teachers.TeacherID",
+          to: "statusUpdates.TeacherID",
+        },
+      },
+    };
   }
 
-  static get jsonSchema () {
+  static get jsonSchema() {
     return {
-      type: 'object',
-      required: ['PhoneNumber', 'FullName', 'StatusID'],
+      type: "object",
+      required: ["PhoneNumber", "FullName", "StatusID"],
```

Comment:
I'm not sure if we should make `TeachingExperience` a `required` input as well? (if we are using it as part of the matching logic in future)

If we decide to make it a required input, then we will also need it to `.defaultTo("No")` in the migrations file 

`
exports.up = function(knex, Promise) {
    return knex.schema.table(tableTeachers, (table) => {
        table.enu('TeachingExperience', ["Yes", "No"]).notNullable().defaultTo("No");
    });
};
`

Reply:
Yup, we should make it a required input! thanks for bringing that up :D i've never made a migration file before.. haha

Reply:
Haha great job Tomoe! Saw the `.notNullable().defaultTo("No")` in the migrations file :)
We can then also update the Teachers model to make TeachingExperience a required below (see below):

```suggestion
  static get jsonSchema() {
    return {
      type: "object",
      required: ["PhoneNumber", "FullName", "StatusID", "TeachingExperience"],
```